From owner-linux-pci@atrey.karlin.mff.cuni.cz Fri Mar 16 19:56:08 2007
Date: Fri, 16 Mar 2007 19:55:52 -0700
From: Randy Dunlap <randy.dunlap@oracle.com>
To: linux-pci@atrey.karlin.mff.cuni.cz
Cc: gregkh <greg@kroah.com>
Subject: PCI: kernel-doc fix
Message-Id: <20070316195552.21188a03.randy.dunlap@oracle.com>

From: Randy Dunlap <randy.dunlap@oracle.com>

Warning(linux-2621-rc3g7/drivers/pci/pci.c:1283): No description found for parameter 'dev'

Signed-off-by: Randy Dunlap <randy.dunlap@oracle.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/pci/pci.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/pci/pci.c
+++ b/drivers/pci/pci.c
@@ -1299,7 +1299,7 @@ pci_intx(struct pci_dev *pdev, int enabl
 
 /**
  * pci_msi_off - disables any msi or msix capabilities
- * @pdev: the PCI device to operate on
+ * @dev: the PCI device to operate on
  *
  * If you want to use msi see pci_enable_msi and friends.
  * This is a lower level primitive that allows us to disable
